src/operation/emulation_model.jl

```diff
@@ -392,14 +395,17 @@ function run_impl(
     if internal.execution_count > 0
         error("Call build! again")
     end
+
     try
         prog_bar =
             ProgressMeter.Progress(internal.executions; enabled = enable_progress_bar)
         for execution in 1:(internal.executions)
-            # timed_log = get_solve_timed_log(model)
-            # _, timed_log[:timed_solve_time], timed_log[:solve_bytes_alloc], timed_log[:sec_in_gc] = @timed
-            one_step_solve!(model)
-            #write_results(model)
+            timed_log = get_solve_timed_log(model)
```

Comment:
we probably need to make this consistent later with the decision model since the timer now captures the solver's time and also the `aux_variable` and duals `calculation` we can extract the "pure" solver time using JuMP.